gempython/tools/amc_user_functions_uhal.py

```diff
@@ -172,3 +172,10 @@ def blockL1A(device):
 
 def enableL1A(device):
     writeRegister(device, "GEM_AMC.TTC.CTRL.L1A_ENABLE", 0x1)
+
+def getAMCL1Acount(device):
+    return readRegister(device, "GEM_AMC.TTC.L1A_ID")
```

Comment:
It's not that it interferes with the DAQ, it's that it's completely coupled to using the DAQ module.
Nothing in the way the firmware scans are run does anything to this module.
* If it is reset when the AMC receives an EC0, this may not be what we want either, as the EC0 could come from upstream when we're not "running"
* As I understand it, this `L1A_ID` register is designed to be used in conjunction with other DAQ module registers, as setting up the AMC to take a "run" will initialize this module and reset everything as needed (something the firmware based scans won't do)
So the question that needs to be answered is: exactly what information is this function supposed to give?